### ISSUE TITLE
support custom endpoints

### ISSUE THREAD

**Issue by hitchhooker** [buffed-labs/dot-console#5]
could you add support for rococo-local/custom-chainspecs?
this seems to me main pain point in using papi/reactive-dot/dot-console right now, not being able to connect to local development environment chains with same methods as production. 

EDIT: just option to write write custom websocket from frontend would probably do now. 
```
// const jsonRpcProvider = WsProvider("wss://some-rpc-endpoint.io")
```

**Comment by tien** [comment]
Yes, many have requested this feature, which I'll have a look at implementing next 👌
